### ISSUE TITLE
Possible bug in binary classification logits in MTL estimator

### ISSUE THREAD

**Issue by joshchuable** [hardianlawi/MTL-Homoscedastic-Uncertainty#2]
In [L118](https://github.com/hardianlawi/MTL-Homoscedastic-Uncertainty/blob/d3aa7e0996f868c6926aa162474d7760a40a0fd7/scripts/estimators.py#L151) it you update the the logit dictionary with `logits[target_name] = logits[target_name]/uncertainties[target_name]`

You later do this again when computing the task specific loss for binary classification
`specific_loss = tf.nn.sigmoid_cross_entropy_with_logits(
                        labels=labels[target_name],
                        logits=logits[target_name]/uncertainties[target_name],
                    )`

**Comment by joshchuable** [comment]
Nevermind... I misread the paper.

**Comment by Cumberbatch08** [comment]
> In [L118](https://github.com/hardianlawi/MTL-Homoscedastic-Uncertainty/blob/d3aa7e0996f868c6926aa162474d7760a40a0fd7/scripts/estimators.py#L151) it you update the the logit dictionary with `logits[target_name] = logits[target_name]/uncertainties[target_name]`
> 
> You later do this again when computing the task specific loss for binary classification
> `specific_loss = tf.nn.sigmoid_cross_entropy_with_logits( labels=labels[target_name], logits=logits[target_name]/uncertainties[target_name], )`

I think if in L118, logits is divided by sigma, it does't need to do agin in loss.
